### DISCUSSION TITLE
Error: /turing/conversation/create: failed to parse response body

### DISCUSSION THREAD

**Discussion by maggieqnr** [danny-avila/LibreChat#1072] [Troubleshooting]
### Contact Details

bing does not work

### What is your question?

After set up the cookies for Bing, it still does not work.

### More Details

![搜狗截图20231018174411](https://github.com/danny-avila/LibreChat/assets/146079448/49505a0f-9e05-4ca4-9f8c-25aca8038fb2)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There could be multiple reasons for this, see here: https://github.com/waylaidwanderer/node-chatgpt-api/issues?q=is%3Aissue+turing
